briefer_closure_syntax.md

```diff
@@ -79,7 +79,7 @@ $result = fn(2);
 
 ```
 
-Exactly what the syntax should be for use`ing variables seems to need to be settled.
```

Comment:
This is archaic....but not an error.

For any command line command, or function in programming, to indicate the precise command that is going to be run, the command is left spelled as it is used, and then "'ing" or "'ed" is used to conjugate it.

e.g. chmod'ed, fopen'ed or in this case use'ing.

I personally prefer backticks for that...but probably a bad idea in a markdown doc.